internal/pkg/aws/s3/s3.go

```diff
@@ -13,26 +13,35 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 )
 
 const (
 	artifactDirName = "manual"
+	batchSize       = 1000
```

Comment:
Do we need this? I think the SDK will automatically reply to us with 1000 items if there are at least that many objects.